versions/3.1.1.md

```diff
@@ -2773,6 +2776,11 @@ For example:
 ```yaml
 components:
   schemas:
+    MyResponseType:
+      oneOf:
+      - $ref: '#/components/schemas/Cat'
+      - $ref: '#/components/schemas/Dog'
+      - $ref: '#/components/schemas/Lizard'
```

Comment:
My understanding is that the `"allOf"` form specifically exists to avoid having to list the subschemas in a `"oneOf"` or `"anyOf"`, so this should not be changed.  I don't personally understand how this even works, and [I'm not the only one](https://github.com/OAI/OpenAPI-Specification/issues/3591).  But it's related to the controversial thing- if all schema names resolve to the Components Object of what we now call the ["entry document"](https://learn.openapis.org/glossary.html), you can just search that one list.  I guess?  Apparently we need to figure that out.

Reply:
The problem I was trying to solve here is that the example payloads never say which schema is being used to evaluate those payloads. Based on my understanding of the `allOf` usage, none of those schemas could produce the results the documentation says it does, so I introduced something that could.

At this point I can only assume that the "Pet" schema is what these payloads are intended to be evaluated against and the behavior described is correct in that case. So, I've reverted my change other than to specify that "Pet" is the schema these payloads are evaluated against. Let me know if that's the right thing to do. I can't see how that makes sense, so I wouldn't be surprised if it's still not correct. Let me know.